### ISSUE TITLE
[Accessibility][Visual Requirement-Create from template]: Luminosity ratio of all templates within 'Create from template' is less than 4.5:1.

### ISSUE THREAD

**Issue by ShellySoni** [microsoft/BotFramework-Composer#2177]
**User Experience:**
Users with low vision or sensitive to low luminosity ratio colors would be impacted as the visibility of templates is less than the standard luminosity ratio of 4.5:1.

**Test Environment:**
Environment: Chromium Edge (Anaheim) + Narrator
Build Version: 2004 (OS Build 19564.1005)​​
Microsoft Edge Version: 82.0.425.0 (Official build) canary (64-bit)​
Test URL: http://localhost:3000/home

**Repro Steps:**
1. Open URL: http://localhost:3000/home
2. Navigate on homepage with tab key up to 'New' button and press enter on it.
3. 'Create from scratch' pop up will open.
4. Select 'Create from Template' radio button and then press tab key to land on its first template i.e. Echo bot.
5. Observe the CCA with color contrast analyzer and find the issue.

**Actual:**
Luminosity ratio of all templates within 'Create from template' is less than 4.5:1.

**Expected:**
Luminosity ratio should be >=4.5:1 for all templates within 'Create from template'.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BA78189A8-A3DC-42C4-A464-A051EBBBD5DF%7D&file=Guideline%2017A%20-%20Contrast%20(Minimum)%20When%20Conveying%20Information.docx&action=default&mobileredirect=true

**Attachment:**
[MAS1.4.3_Create From template.zip](https://github.com/microsoft/BotFramework-Composer/files/4292931/MAS1.4.3_Create.From.template.zip)

**Comment by corinagum** [comment]
Can we get the attachments added in? Thank you.

**Comment by Padmini171991** [comment]
![MAS1 4 3_Create From template](https://user-images.githubusercontent.com/60918692/76081080-5f00a180-5fce-11ea-9fd6-1f3c84dcf172.png)

Attaching screenshot here. Somehow the zip file got dragged on top of User Experience.

**Comment by corinagum** [comment]
@Padmini171991 oops, I see it now, thank you!. Moved it to bottom of the post.

**Comment by DesignPolice** [comment]
Thanks for the Tag @corinagum, I was wondering when this one would come up. It is a hack of a old design, that was altered just to get things moving. Let me find out if they want to alter the hack, or implement an actual fix that we have.

I will touch back ASAP

**Comment by corinagum** [comment]
Related to #1728 , so there will probably be a decent amount of discussion before a spec is ready.

**Comment by DesignPolice** [comment]
Thanks Corina - @cwhitten and @mareekuh - Lets talk about this in Stand Up today if we have time or we have someone waiting to fix it.

**Comment by corinagum** [comment]
@DesignPolice roger that, will do. Thanks for checking up!

**Comment by awalia13** [comment]
Hi Corina ,We will validate this bug and share update asap.

**Comment by ashish315** [comment]
As checked, still the issue is not fixed.

**Comment by corinagum** [comment]
I have done a fresh build of Composer and verified this fix. Please re-verify on your side.

**Comment by ashish315** [comment]
@corinagum Checked on the latest build and the issue is resolved. So, HCL is closing the issue.
Current CCA result-
Foreground:#000000
Background:#EBEBEB

The contrast ratio is: 17.6:1
